grails-app/i18n/messages.properties

```diff
@@ -592,7 +592,7 @@ documentType.label                           = Document Type
 email.errorReportSubject.message				= Bug Report
 email.errorReportSuccess.message 				= Your error has been reported to {0}
 email.errorReportDisabled.message 				= The bug reporting system has been disabled. Please contact your system administrator for more information.
-email.sent.message								= Email was successfully sent to {0}
+email.sent.message								= Email was successfully sent

```

Comment:
I think this message is also used in user controller, so you need to change it there too